fix: MD034 Bare URLs (#20020)
mdn/content: files/en-us/learn/javascript/asynchronous/introducing_workers/index.md

> **Note:** To run this site, you'll have to run a local web server, because file:// URLs are not allowed to load workers. See our guide to [setting up a local testing server](/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server). With that done, you should be able to click "Generate primes" and have your main page stay responsive.
>

> > If you have any problems creating or running the example, you can have a look at the [finished version](https://github.com/mdn/learning-area/blob/main/javascript/asynchronous/workers/finished) and try it [live](https://mdn.github.io/learning-area/javascript/asynchronous/workers/finished).

Reviewer: I would suggest reverting to "see"

Reviewer 2: ```suggestion
> If you have any problems creating or running the example, you can review the [finished version](https://github.com/mdn/learning-area/blob/main/javascript/asynchronous/workers/finished) and try it [live](https://mdn.github.io/learning-area/javascript/asynchronous/workers/finished).
```
